mw-webapp/src/component/modal/Modal.stories.tsx

```diff
@@ -9,31 +11,35 @@ const meta = {
 };
 
 export default meta;
-type Story = StoryObj<typeof meta>;
-
-const defaultTrigger = (
-  <button>
-    Click me!
-  </button>
-);
 
 const defaultContent = (
   <div>
     Lorem ipsum dolor sit amet, consectetur adipisicing elit...
   </div>
 );
 
-export const Default: Story = {
-  args: {
-    trigger: defaultTrigger,
-    content: defaultContent,
-  },
-};
+const Template: StoryFn<typeof Modal> = (args) => {
+  const [open, setOpen] = useState(args.open);
 
-export const Opened: Story = {
-  args: {
-    trigger: defaultTrigger,
-    content: defaultContent,
-    open: true,
-  },
+  return (
+    <>
+      <Button
+        value="Click me!"
+        onClick={() => {
+          setOpen(true);
+        }}
+      />
+      <Modal
+        content={defaultContent}
+        open={open}
+        setOpen={setOpen}
```

Comment:
how does it possble that both (isOpen, setOpen) params exist?

Reply:
What do you mean? Only two params exist, state value and set state function